mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -374,6 +380,22 @@ private[ann] trait TopologyModel extends Serializable {
   def predict(data: Vector): Vector
 
   /**
+   * Raw prediction of the model
```

Comment:
This documentation does not add any information.  Can you please link to ProbabilisticClassifier instead?